gateway/main/settings.py

```diff
@@ -28,6 +28,10 @@
 
 ALLOWED_HOSTS = os.environ.get("ALLOWED_HOSTS", "*").split(",")
```

Comment:
@pacomf @Tansito related to our earlier discussion, I think we can use something like this in the repository to allow host configuration. will work on that tomorrow